### PR TITLE
Added info about Fastly Admin UI Not Being Available (Cloud)

### LINKED ISSUE #7892: How to get access to the Fastly Admin UI?
# General issue

Page: /cloud/cdn/configure-fastly.html

## Description:

Some merchants have been asking how to get access to the Fastly Admin UI.

The article currently mentions this:
"You need the Fastly credentials to configure Fastly CDN services from the Magento Admin UI and to submit Fastly API requests."

For the sake of clarity, I think it should emphasize/add the caveat that we do not provide Fastly Admin UI access. 

## Possible solutions:

I'm not sure whether/how this information should be posed in the article, but the reason for that is the Fastly account is owned by Magento with sub-accounts created for each merchant. 

However, there's no "role management/segregation" feature available within the UI, which means that any user who has been given access to the Admin UI would have the ability to access the sub-accounts.

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

This pull request (PR) fixes #7892 

## Affected DevDocs pages

https://devdocs.magento.com/cloud/cdn/configure-fastly.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/cdn/configure-fastly.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ During project provisioning, Magento adds your project to the [Fastly service ac
 
 You need the Fastly credentials to configure Fastly CDN services from the Magento Admin UI and to submit Fastly API requests.
 
+{:.bs-callout-info}
+With Magento Cloud, Fastly Admin UI access is not provided. Any issue that cannot be solved through the Fastly Admin in Magento should be referred to [Magento Support](https://support.magento.com/hc/en-us/articles/360000913794#submit-ticket).
```
Comment (meker12) [inline src/cloud/cdn/configure-fastly.md:32]:
```suggestion
With {{ site.data.var.ece }}Magento Cloud, you cannot access the Fastly Admin UI directly. You must use the Magento Admin UI to review and update the Fastly configuration for your environments.  If you cannot resolve an issue using the Fastly capabilities in the Magento Admin UI, submit a [Magento Support](https://support.magento.com/hc/en-us/articles/360000913794#submit-ticket) ticket.
```

**Review by meker12 — CHANGES_REQUESTED** [review]
Added suggested clarifications.

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**